src/app.js

```diff
@@ -24,6 +24,7 @@ app.use(express.static('public'))
 app.use(favicon(__dirname + '/public/favicon.ico'));
 
 // routes
-app.use('/api/v1', mainRouter, userRouter, classesRouter);
+app.use('/api/v1', mainRouter, classesRouter);
```

Comment:
classRouter is here and on the next line))) Where it should 'live"?

Reply:
done . in users should be only userRoutes